docs/book/introduction.md

```diff
@@ -16,15 +16,135 @@ description: Welcome to ZenML!
-* With these abstractions, it remains **extensible** and **open to customization**. That is also backed by numerous **collaborative features** that allow you to work not just with your team but also with the community.
-* Moreover, this creates a separation between the different aspects of ML workflows, allowing users to focus on creating their own respective solutions in an efficient manner.
-* Overall, it reduces the complexity of your development lifecycle, all the way **from an experimentation environment to a production-grade solution**.
+**ZenML** is an extensible, open-source MLOps framework for creating portable, 
+production-ready MLOps pipelines. 
+
+By decoupling infrastructure from code, ZenML gives all ML developers in your 
+organization more freedom and independence in how they approach their work:
+
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
+
+{% tabs %}
+
+{% tab title="ZenML for Data Scientists" %}
+
+As a Data Scientists, ZenML gives you the freedom to focus on modeling and
+experimentation without having to worry about production issues at all.
+
+**Tooling Freedom:** Since ZenML code runs in any environment, you can 
+develop ML models locally with all your favourite tools. Once you are happy with 
+your results, you can switch to a production environment via a single command 
+and all your code will just work:
+
+```bash
+python run.py  # develop your code locally with all your favourite tools
+zenml stack set production
+python run.py  # run your code in production without any code changes
+```
+
+**Pythonic SDK:** ZenML is designed to be as unintrusive as possible. Adding 
+a ZenML `@step` or `@pipeline` decorator to your Python functions is enough to 
+turn your existing code into ZenML pipelines:
+
+```python
+@step
+def step_1() -> str:
+  return "world"
+
+@step
+def step_2(input_one: str, input_two: str) -> None:
+  combined_str = input_one + ' ' + input_two
+  print(combined_str)
+
+@pipeline
+def my_pipeline():
+  output_step_one = step_1()
+  step_2(input_one="hello", input_two=output_step_one)
+
+my_pipeline()
+```
+
+**Reusability:** Defining your code as ZenML steps brings standardization 
+into your organization, allowing you to easily reuse code developed by your 
+colleagues:
+
+```python
+from my_organization.steps import data_loader_step, model_validation_step
+
+@step
+def my_model_trainer_step(data):
+    ...
+
+@pipeline
+def my_pipeline():
+  data = data_loader_step()
+  model = my_model_trainer_step(data)
+  eval_metric = model_validation_step(model)
+```
+{% endtab %}
+
+{% tab title="ZenML for ML Engineers" %}
+
+As an ML Engineer, ZenML enables you take ownership of the entire ML workflow
+end-to-end. Adopting ZenML means less handover points and more visibility on
+what is happening in your organization.
+
+**ML Lifecycle Management:** ZenML's abstractions make it possible to manage the
```

Comment:
Not sure if this is possible but can we add an image/code block/CLI commands somewhere in the ML Engineer tab. In comparison to the other tabs, this tab looks a bit too text-heavy.